contrib/test_jenkins.sh

```diff
@@ -413,11 +413,19 @@ run_uct_hello() {
 #
 run_ucx_perftest_mpi() {
 	ucx_inst_ptest=$ucx_inst/share/ucx/perftest
+	run_cuda_test=0
 
 	# hack for perftest, no way to override params used in batch
 	# todo: fix in perftest
 	sed -s 's,-n [0-9]*,-n 1000,g' $ucx_inst_ptest/msg_pow2 | sort -R > $ucx_inst_ptest/msg_pow2_short
-	cat $ucx_inst_ptest/test_types | sort -R > $ucx_inst_ptest/test_types_short
+
+	cat $ucx_inst_ptest/test_types | grep -v cuda | sort -R > $ucx_inst_ptest/test_types_short
+	#check if cuda tests have to run
+	if (lsmod | grep -q "nv_peer_mem") && (lsmod | grep -q "gdrdrv")
+	then
+		run_cuda_test=1
+		cat $ucx_inst_ptest/test_types | grep cuda | sort -R > $ucx_inst_ptest/test_types_short
```

Comment:
it looks like if cuda is loaded, it will run *only* cuda tests?
also, is the mlx5 restriction really needed? if this is not IB, UCP could still use gdrcopy/pipeline, right?